src/components/ViewRoutingSlip/LinkedRSDetails.vue

```diff
@@ -0,0 +1,43 @@
+<template>
```

Comment:
We may need some consistency in file naming, either use `RS` across all files, or use `RoutingSlip`